internal/services/events_test.go

```diff
@@ -108,26 +116,165 @@ func Test_DelegationExpansion(t *testing.T) {
 
 	btcNotifier := mocks.NewBtcNotifier(t)
 	// delegation spend notification registration
-	stakingTxHash, err := chainhash.NewHashFromStr(stakingTxHashHex)
-	require.NoError(t, err)
+	stakingTxHash := hashFromString(t, stakingTxHashHex)
+	expansionStakingTxHash := hashFromString(t, expansionStakingTxHashHex)
 
-	outpoint := &wire.OutPoint{
+	stakingOutpoint := &wire.OutPoint{
 		Hash: *stakingTxHash,
 	}
-	btcNotifier.On("RegisterSpendNtfn", outpoint, mock.Anything, startHeight).Return(&chainntnfs.SpendEvent{}, nil).Once()
+	delegationSpendCh := make(chan *chainntnfs.SpendDetail, 1) // we need unbuffered because there is no reader yet
+	delegationSpendCh <- &chainntnfs.SpendDetail{
+		SpentOutPoint: stakingOutpoint,
+		SpenderTxHash: expansionStakingTxHash,
+		SpendingTx: &wire.MsgTx{
+			Version: 2,
+			TxIn: []*wire.TxIn{
+				{
+					PreviousOutPoint: *stakingOutpoint,
+					SignatureScript:  []byte{},
+					Witness: [][]byte{
```

Comment:
I wrote a script to get all this data (notifier couldn't get it). 
Right now I'm not sure which is better: keep this data here or move it somewhere